cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -142,7 +142,8 @@ bool CobTwistController::initialize()
     {
         twist_controller_params_.frame_names.push_back(chain_.getSegment(i).getName());
     }
-    register_link_client_ = nh_.serviceClient<cob_srvs::SetString>("obstacle_distance/registerLinkOfInterest");
+
+    register_link_client_ = nh_.serviceClient<cob_srvs::SetString>("/register_links");
```

Comment:
Do not use absolute namespaces here. You can remap the namespaces inside the launch file if you need to.